client/src/pages/Profile/Profile.jsx

```diff
@@ -475,30 +480,54 @@ ProfilePageScuntToken.propTypes = {
 
 const ProfilePageLeaderPermissionDashboardLinks = () => {
   const { user } = useSelector(userSelector);
+  const { darkMode, setDarkModeStatus } = useContext(DarkModeContext);
+  const [openDataDropdown, setOpenDataDropdown] = useState(true);
```

Comment:
set the initial state of the dropdowns to false, the dropdowns seem to be initially open
![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/86009011/7e603d4f-4daf-422b-99d6-90986b6adb67)